firstvoices/backend/serializers/utils.py

```diff
@@ -1,9 +1,32 @@
 import logging
 
+from rest_framework import serializers
 from rest_framework.exceptions import APIException
 
 from backend.models.sites import Site
 
+# For import-jobs header validation
```

Comment:
Since these functions are only going to be used in one place, lets not put them in the reusable utils that get imported multiple places. Either keep them in import_job_serializers or make a standalone file like csv_utils